bluesky/plans.py

```diff
@@ -81,6 +81,7 @@ def __iter__(self):
         for token in tokens:
             yield Msg('unsubscribe', token)
         yield from self._post()
+        yield Msg('checkpoint')
```

Comment:
> is planbase otherwise emitting checkpoints?

No, but all the mid-level base classes (Plan1D, PlanND, _AdaptivePlanBase) do, so all the plans do.

> Do we need to add a clear_checkpoint message?

Yes.